waifus/nezuko.json

```diff
@@ -0,0 +1,11 @@
+{
+	"type" : 0,
+	"english-name" : "Nezuko Kamado",
+	"image" : "https://vignette.wikia.nocookie.net/kimetsu-no-yaiba/images/2/2f/Nezuko_anime_design.png/revision/latest?cb=20181128204224",
+	"japanese-name" : "竈門 禰豆子",
+	"month" : 12,
+	"day-of-month" : 28,
+	"year" : 2005,
```

Comment:
I don't see anywhere in the sources that say that she was born in 2005.

Reply:
A friend of mine told me, that the show doesn't have a year where it takes place in. So 2005, or any year isn't correct.

Reply:
Sorry for the late reply, yeah because they only put her current age is 14 and now is 2019 ? so I just get 2005 from that ... So better to remove that year and replace it with `
  "definitely-legal": false,` ?

Reply:
No :<

Reply:
So ... what should I do ?

Reply:
I would use the age in show variable https://github.com/yourWaifu/is-your-waifu-legal/blob/master/reference.md#story-variables

Reply:
thanks for your input, done :)